src/app/Main.java

```diff
@@ -118,7 +127,22 @@ public static void main(String[] args) {
         MainPageView mainPageView = new MainPageView(generateRecipeController,cookThisOrReRollViewModel,redirectToPreferenceController);
         views.add(mainPageView, mainPageView.viewName);
 
-        viewManagerModel.setActiveView(mainPageView.viewName);
+        loginViewModel LVM = new loginViewModel();
+        userDataAccessObject UDAO = new userDataAccessObject();
+        loginPresenter LoginPresenter = new loginPresenter(LVM, viewManagerModel, "main");
+        LoginInteractor loginInteractor = new LoginInteractor(UDAO, LoginPresenter, inMemoryDataAccessUser);
+        loginController LoginController = new loginController(loginInteractor);
+
+        signUpViewModel SVM = new signUpViewModel();
+        signUpPresenter signUpPresenter = new signUpPresenter(SVM, mainPageView.viewName, viewManagerModel);
+        signUpInteractor signUpInteractor = new signUpInteractor(UDAO, signUpPresenter, inMemoryDataAccessUser);
+        signUpController signUpController = new signUpController(signUpInteractor);
+        signUpView SignUpView = new signUpView(signUpController, SVM);
+        views.add(SignUpView, SVM.getViewName());
+        LoginView loginView = new LoginView(LVM, LoginController, signUpController);
+        views.add(loginView, LVM.getViewName());
+
+        viewManagerModel.setActiveView(loginView.viewName);
```

Comment:
- Nice that you are incorporating your login in and sign up use cases in the main function
- Right now this is good, but later we might want to refactor the code into a factory class  
- At the end you set the initial view to loginView. Nice